docs/src/snap/howto/networking/alternative-cni.md

```diff
@@ -0,0 +1,107 @@
+# How to use an alternative CNI
+
+While {{product}} ships with a default [Container Network Interface] (CNI) that
+we guarantee is compatible with our distribution, you may want to use a
+different CNI plugin for your specific networking requirements. This guide
+explains how to safely replace the default CNI with an alternative solution.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You have root or sudo access to the machine.
+- Basic understanding of Kubernetes networking concepts.
+- Basic knowledge of Helm.
+
+## Disable default network implementation
+
+For an existing cluster, disable the default network
+plugin:
+
+```
+sudo k8s disable ingress gateway network
```

Comment:
Should we add a note that the underlying implementation ships all three features and you can't just disable one?